kitsune/sumo/static/sumo/kb-ux-experiment/src.682efaae.js

```diff
@@ -0,0 +1,375 @@
+                breadcrumbs.classList.remove('active');
+
+                hideAll(containers);
+            }
+        }, false);
+
+        initColumns(containers, liWidth, columnLength, dropMenuClass, '.' + stepClass);
+
+        window.addEventListener('resize', function () {
+            initColumns(containers, liWidth, columnLength, dropMenuClass, '.' + stepClass);
+        });
+
+        function initColumns(list, columnWidth, maxColumnItemsCount, dropMenuCls, stepCls) {
+            for (var i = 0; i < list.length; i++) {
+                var dropMenu = list[i].querySelector(dropMenuCls);
+                var step = list[i].querySelector(stepCls);
+                var windowWidth = window.innerWidth;
+
+                if (dropMenu !== null && step !== null) {
+                    if (windowWidth > query) {
+                        var wantedColumns = Math.ceil(dropMenu.childElementCount / maxColumnItemsCount);
+                        var available = Math.floor((windowWidth - step.getBoundingClientRect().left) / columnWidth);
+                        var state = Math.min(wantedColumns, available);
+
+                        state = state > 2 ? 3 : state;
+
+                        dropMenu.setAttribute('data-columns', state.toString());
+                    }
+                }
+            }
+        }
+    })(window);
+
+    // video slider
+    (function (Swiper) {
+        if (typeof Swiper !== 'function') return;
+
+        var thumbs = document.querySelector('.gallery-thumbs');
+        var galleryTop = document.querySelector('.gallery-top');
+
+        if (thumbs !== null && galleryTop !== null) {
+            var thumbsSlider = new Swiper(thumbs, {
+                spaceBetween: 10,
+                slidesPerView: 'auto',
+                // freeMode: true,
+                watchSlidesVisibility: true,
+                watchSlidesProgress: true,
+                on: {
+                    click: function click() {
+                        var videos = galleryTop.querySelectorAll('video');
+
+                        for (var i = 0; i < videos.length; i++) {
+                            videos[i].pause();
+                        }
+                    }
+                }
+            });
+
+            var galleryTopSlider = new Swiper(galleryTop, {
+                spaceBetween: 10,
+                wrapperClass: 'gallery-wrapper',
+                slideClass: 'gallery-slide',
+                slidesPerView: 1,
+                allowTouchMove: false,
+                thumbs: {
+                    swiper: thumbsSlider
+                }
+            });
+        }
+    })(window.Swiper);
+}
+
+function ready(fn) {
+    if (document.attachEvent ? document.readyState === "complete" : document.readyState !== "loading") {
+        fn();
+    } else {
+        document.addEventListener('DOMContentLoaded', fn);
+    }
+}
+
+ready(fn);
+},{}],16:[function(require,module,exports) {
+'use strict';
+
+require('./styles/reset.scss');
```

Comment:
What is happening with all of this `require()` and direct use of `scss` files? Is this rendering SCSS in the browser? We should not be doing that if so. We have a static asset pipeline that will pre-render this stuff.

Reply:
I believe currently it's ignored completely. This file is the output of the node build. Even the path here is wrong. I will include the native js version in the pipeline. Thanks for catching this!